doc/locale/ja/LC_MESSAGES/reference.po

```diff
@@ -30176,3 +30198,13 @@ msgstr ""
 
 msgid "``window_sum``"
 msgstr ""
+
+#~ msgid "There is one required parameter ``column``."
+#~ msgstr "必須引数が一つあり、それは ``column`` です。"
+
+#~ msgid "Here is query to show round value of ``timestamp`` to a day unit."
+#~ msgstr ""
+#~ "これは、 ``timestamp`` の値を日単位に丸めた値を表示するクエリーです。"
+
+#~ msgid "``time_classify_day`` has one parameter::"
+#~ msgstr "``time_classify_day`` の引数は1つだけです::"
```

Comment:
Could you remove unused translations?

Reply:
Ah...I had overlooked them.
I've removed them.